### ISSUE TITLE
Importing old format suite

### ISSUE THREAD

**Issue by lcsqlpete** [SeleniumHQ/selenium-ide#188]
My issue on opening test cases/suites was closed so continuing with a new post.

As mentioned, I was able to open my old format test cases by renaming the files to have an .html extension but still a couple of issues.

It appears I can only have 1 test case file open at a a time when Tests is selected in the dropdown menu. If I open a test case when one is already opened, the original one disappears and the new one is opened.  I can't remember for sure but I don't think that was the case in the old version of Selenium.

I ran into several issues trying to import an old format test suite. 

The old format test suite file refers to the files without the .html extension so the ones with the .html extension aren't recognized when I drag drop them onto the import dialog.  I fixed that by editing the test suite file to include the .html extension to all the test case file names. I was then able to import them.

One test case file would not import, no error message just wouldn't drop or open by selecting from the browse link. I edited the test suite file to remove the one that would not import, after which I was able to successfully import the test suite file.  I have attached the offending test case file (Curtains.html) inside the zip file.

There are at least three commands that don't appear to be supported in the new Selenium: assertLocation, verifyLocation, and storeTable (those are the old names).  They were not converted to a name that has spaces in it and the test cases fail on them when executed.  They also don't appear in the command field when I start typing them in.

Two of the test cases were imported with a name of "New Test" instead of their original name and one was imported with some of the characters changed from uppercase to lowercase..

Final problem is that the Test suite lists the test cases in alphabetical order and tries to execute them in that order which won't work at all.  I tried to drag/drop them into the correct order but the drop part of it doesn't work.I have attached the test suite file (WebApp_QuickLook_KEA.html) in the zip file.

Hopefully these issues can be solved otherwise I wont be able to use the new Selenium.

[Archive.zip](https://github.com/SeleniumHQ/selenium-ide/files/1975417/Archive.zip)

**Comment by corevo** [comment]
I'm gonna break this down to it's components, because the way it is written makes it hard to follow.  
- Firstly test cases and suites need to end with `.html` to be imported, as well as new ones end with `.side`.  
- Opening a test case over a test case causes the previous one to close, perhaps we can change that behavior.
- I will take a further look at the test case that wasn't able to be imported, my guess is that it's a bug with parsing, the old IDE didn't follow the spec of `xhtml` meaning there were a few bugs in the way it saved test cases, and we have to find them and fix them.  
- The three commands that you've mentioned aren't available in the new IDE, please post them on their own issue, we add commands per request, since it would take us months to add all of them.  
- Can you tell me which files were renamed to "New Test", my guess is that they don't have a title.  
- The test case list order is a duplicate of #176, please follow it on that thread.

**Comment by corevo** [comment]
The curtain test case failure originates from the fact that the name is incorrect.  
The file name is `Curtains.html` while the name inside the suite is `AssignCurtains.html`.  
You must have renamed it at some point, hence the new IDE can't pick it up.

**Comment by lcsqlpete** [comment]
Thanks Tomer.  I will add a new issue for the missing comands.

Pete

**Comment by lcsqlpete** [comment]
The files which were imported as "New Test" were BadLogin.html and
KEALogin.html.  They happen to be the first two files named in the suite.

The file that was imported with upper/lowercase issues was AddKEA,html
which was imported as Addkea.

Thanks,
Pete

**Comment by lcsqlpete** [comment]
I just noticed that the file EditUser was imported twice.

**Comment by corevo** [comment]
Can you attach them?  
The zip only contains Curtains.html

**Comment by tourdedave** [comment]
Closing this issue due to prolonged inactivity.
